Header header           # 

float32 angleRad        # Angle to center lane in radians
float32 distNormalized  # Distance to center lane [-1,1]. Normalized to lane width (-1 = leftlane, ..)
float32 velocityMS      # Current velocity in m/s
